tests/plugins/tool/black_tool_plugin/valid_package/format_errors.py

```diff
@@ -1 +1 @@
-x =  "some string",
+x = ("some string",)
```

Comment:
Probably don't want to change formatting on this file. Was likely intentional to make sure our tools catch legitimate errors.